packages/miniapp-registry/src/index.ts

```diff
@@ -9,12 +9,14 @@ import NFTMinter from "@miniapps/nft-minter";
 import UrlRender from "@miniapps/url-render";
 import ImageRender from "@miniapps/image-render";
 import ChatGPTShorten from "@miniapps/chatgpt-shorten";
+import ZoraNftMinter from "@miniapps/zora-nft-minter";
 
 export const allMiniApps = [
   InfuraIPFSUpload,
   LivepeerVideo,
   GiphyPicker,
   VideoRender,
+  ZoraNftMinter,
   NFTMinter,
```

Comment:
ZoraNFTMinter and NFTMinter will both be triggered by NFT urls. We should confirm that the zora one is getting preference

Reply:
Currently the priority is determined by this array (earlier index gets matched first)